src/worker/BBBLiveStream.ts

```diff
@@ -132,8 +132,38 @@ export class BBBLiveStream{
 
         await this.page.goto(this.joinUrl);
 
+    
+        // Find listen only button
+        let listenOnlyLocator = '';
+
+        const isMeteor = await this.page.evaluate(() => {
+            return eval('typeof __meteor_runtime_config__') === 'object';
+        });
+        if(isMeteor){
+            // BBB <= 2.7
+            listenOnlyLocator = '[data-test="listenOnlyBtn"]';
+        }
+        else{
+            // BBB >= 3.0
+
+            // Wait for window.meetingClientSettings to be available, wait 10 sec.
+            const clientSettings = await this.getObject('window.meetingClientSettings', 10000);
```

Comment:
This setting also exists in the old meteor setting and should cover both the BBB 2.7 and 3.0 case.

Reply:
Not on my BBB 2.7.16